src/dev/Core/EastSeat.ResourceIdea.Application/Features/WorkItems/Commands/UpdateWorkItemCommand.cs

```diff
@@ -0,0 +1,67 @@
+using EastSeat.ResourceIdea.Domain.Employees.ValueObjects;
+using EastSeat.ResourceIdea.Domain.Engagements.ValueObjects;
+using EastSeat.ResourceIdea.Domain.Enums;
+using EastSeat.ResourceIdea.Domain.Tenants.ValueObjects;
+using EastSeat.ResourceIdea.Domain.Types;
+using EastSeat.ResourceIdea.Domain.WorkItems.Models;
+using EastSeat.ResourceIdea.Domain.WorkItems.ValueObjects;
+
+using MediatR;
+
+namespace EastSeat.ResourceIdea.Application.Features.WorkItems.Commands;
+
+/// <summary>
+/// Represents a command to update a work item.
+/// </summary>
+public sealed class UpdateWorkItemCommand : IRequest<ResourceIdeaResponse<WorkItemModel>>
```

Comment:
Use `BaseRequest<WorkItemModel>` instead of `IRequest<ResourceIdeaResponse<WorkItemModel>>`

Reply:
I have updated the Command to inherit from the `BaseRequest<ResourceIdea<T>>` class.